fastlane/lib/fastlane/actions/changelog_from_git_commits.rb

```diff
@@ -6,15 +6,22 @@ module SharedValues
 
     class ChangelogFromGitCommitsAction < Action
       def self.run(params)
-        if params[:between]
-          from, to = params[:between]
-        else
-          from = Actions.last_git_tag_name(params[:match_lightweight_tag], params[:tag_match_pattern])
-          UI.verbose("Found the last Git tag: #{from}")
-          to = 'HEAD'
+        if params[:between] and params[:commits_count]
```

Comment:
Use the[ conflicting_option param](https://github.com/fastlane/fastlane/blob/master/fastlane_core/lib/fastlane_core/configuration/config_item.rb#L17) to do this validation